app/views/headings/show.html.erb

```diff
@@ -1,19 +1,14 @@
 <% content_for :title, goods_nomenclature_title(@heading) %>
 
-
 <% content_for :head do %>
   <meta name="description" content="<%= @heading %>">
   <link rel='alternate' type='application/json' href='<%= heading_url(@heading, format: :json) %>' title='Heading information page in JSON format' />
 <% end %>
-
-<% if @heading.declarable? %>
```

Comment:
this heading.declarable? is never reached,
since if it is a declarable heading, the controller would redirect it to the commodity view.

Reply:
I'm thinking this assertion is might be wrong? Doesn't a declarable heading render `show`, but then it renders the same `measures` partials as `commodities/show` ?

Reply:
Looks like this is rendering the declarable content on line 25 of this file?

Reply:
Looking at the helper method which drives this - if you supply a nil header _but_ do supply a block then it will render just the caption + block which is what you need for this I think?

Reply:
but I am happy to move it to a different PR!

Reply:
In the end, I thought it was OK to do It in this PR but on a different commit.